### PR TITLE
fix: fix typescript definitions

### PR DESCRIPTION
**Pull request by kagawagao**

### CODE REVIEW

File: lib/index.d.ts [hunk 1]
```diff
@@ -1,3 +1,3 @@
-import type { Syntax } from 'postcss';
-const postcssHtml: Syntax;
+import type { Syntax } from "postcss";
+const postcssHtml: (options?: Record<string, any>) => Syntax;
```
Comment (ota-meshi) [inline lib/index.d.ts:2]: It is also a function and has Syntax properties. Could you please define a more correct type?

**Review by ota-meshi — CHANGES_REQUESTED** [review]

File: lib/index.d.ts (hunk 1 above)
Comment (kagawagao) [reply lib/index.d.ts]: done

**Review by ota-meshi — APPROVED** [review]
Thank you!

File: lib/index.d.ts [hunk 2]
```diff
@@ -1,3 +1,8 @@
-import type { Syntax } from 'postcss';
-const postcssHtml: Syntax;
+import type { Syntax } from "postcss";
+
+type PostcssHtml = {
```
Comment (edisonLzy) [inline lib/index.d.ts:3]: @kagawagao 这是什么写法 ？ 🤔️

**Pull request merged**